duckduckgo/Android
# Bump Google libraries

Task/Issue URL: https://app.asana.com/0/414730916066338/1199111180919669/f
Tech Design URL: 
CC: 

**Description**:
This PR updates all google libraries to the latest stable version available.
This includes:

- WorkManager
- Material design
- Android x
- Swipe refresh layout component
- AndroidKtxCore
- FragmentKtx
- ContraintLayout
- Webkit

**Steps to test this PR**:
Smoke test: Navigate through the app and ensure no breakages are introduced. 

- Workmanager: Change `Notification.scheduleInactiveUserNotifications` to use `Minutes` instead of hours in order to test notifications.
- UI: run a fresh install and ensure onboarding CTAs are not broken. Also visit other screens like privacy dashboard and feedback flow.
- Swipe refresh layout: ensure pull to refresh works.


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
